mdsobjects/python/tree.py

```diff
@@ -874,6 +874,16 @@ def getVersionDate():
             raise _exc.MDSplusException(status)
         return _scr.Uint64(dt.value).date
 
+    @staticmethod
```

Comment:
I have a way of implementing methods that can be used as static and as instance base on the context it is used. check out ther usage of @staticmethodX used by e.g. compressDatafile